force-app/main/default/classes/DMLOperations.cls

```diff
@@ -31,13 +31,19 @@ public with sharing class DMLOperations {
      */
     public static Id insertNewAccount() {
         // Create a new Account instance
+        Account newAcc = new Account();
 
         // Set some fields on the Account
+        newAcc.Name = 'Test Account AB';
```

Comment:
Try using the Math.random to make sure that different accounts have different names, even if it's a number